tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+ITEMS_TO_SKIP = ['chewed gum', 'rubbish', 'used tissue']
+MAX_INVENTORY_CAP = 80
+HEAVY_WEIGHT_THRESHOLD = 70
+LIGHT_WEIGHT_THRESHOLD = 60
+
+
+def display_inventory(inventory):
+    '''
```

Comment:
Please use triple double quotes docstrings. https://www.python.org/dev/peps/pep-0257/
You can scan your code with `pydocstyle` and it should point out such issues from what I remember.